test/sample_consensus/test_sample_consensus_quadric_models.cpp

```diff
@@ -636,7 +719,7 @@ TEST (SampleConsensusModelCircle3D, RANSAC)
   EXPECT_NEAR (-3.0, coeff[2], 1e-3);
   EXPECT_NEAR ( 0.1, coeff[3], 1e-3);
   EXPECT_NEAR ( 0.0, coeff[4], 1e-3);
-  EXPECT_NEAR (-1.0, coeff[5], 1e-3);
+  EXPECT_NEAR ( 1.0, coeff[5], 1e-3);
```

Comment:
Let's put `std::abs(coeff[5])` here? Same below for refined. For other seeds, the algorithm could as well return `[0, -1, 0]`  as a normal, which would also be valid.

Reply:
Any preferences how to align them after `std::abs` is in place? Or are you fine with a little "bump"?

```cpp
  EXPECT_NEAR ( 1.0, coeff[0], 1e-3);
  EXPECT_NEAR ( 5.0, coeff[1], 1e-3);
  EXPECT_NEAR (-3.0, coeff[2], 1e-3);
  EXPECT_NEAR ( 0.1, coeff[3], 1e-3);
  EXPECT_NEAR ( 0.0, coeff[4], 1e-3);
  EXPECT_NEAR ( 1.0, std::abs (coeff[5]), 1e-3);
  EXPECT_NEAR ( 0.0, coeff[6], 1e-3);
```

Reply:
Yeah, that's fine :+1: